lib/ansible/galaxy/collection/__init__.py

```diff
@@ -1427,6 +1555,8 @@ def install_src(collection, b_collection_path, b_collection_output_path, artifac
         b_collection_path,
         collection_meta['namespace'], collection_meta['name'],
         collection_meta['build_ignore'],
+        collection_meta.get('manifest_directives', Sentinel),
+        collection_meta.get('manifest_directives_full', False),
```

Comment:
Looks good to me.

These should be added to lib/ansible/galaxy/data/collections_galaxy_meta.yml too (so using the options doesn't give a warning).

Reply:
I just moved that `manifest_directives_full` to a `manifest_control` dict. So you caught me at a good time.  I've updated that file.  I think I got what was needed.